npm/webpack-preprocessor/index.ts

```diff
@@ -249,7 +249,7 @@ const preprocessor: WebpackPreprocessor = (options: PreprocessorOptions = {}): F
       webpackOptions.module.rules.unshift({
         test: /\.(js|ts|jsx|tsx)$/,
         use: [{
-          loader: path.join(__dirname, 'lib/cross-origin-callback-loader'),
+          loader: require.resolve('@cypress/webpack-preprocessor/dist/lib/cross-origin-callback-loader.js'),
```

Comment:
If we need to do require.resolve() within the NPM module, is there risk that custom webpack configuration will be problematic or is this only for snapshot resolution in the binary?

Reply:
& vite doesn't have issues because we don't ship with the binary? https://github.com/cypress-io/cypress/blob/develop/npm/vite-dev-server/src/resolveConfig.ts#L14